fast64_internal/oot/scene/exporter/to_c/scene_table_c.py

```diff
@@ -15,7 +22,13 @@ def getSceneTable(exportPath):
         with open(os.path.join(exportPath, "include/tables/scene_table.h")) as fileData:
             # keep the relevant data and do some formatting
             for i, line in enumerate(fileData):
-                if not line.startswith("// "):
+                if (
+                    line != "\n"
+                    and '#include "config.h"\n' not in line
+                    and "#ifdef INCLUDE_TEST_SCENES" not in line
+                    and "#endif" not in line
```

Comment:
I didn't realize but this is HackerOoT-specific code in the middle of generic-decomp code

Reply:
yeah well, I don't really have the choice